src/Sentry/Android/Sentry.Android.props

```diff
@@ -0,0 +1,45 @@
+<Project>
+  <PropertyGroup>
+    <SupportedOSPlatformVersion>21</SupportedOSPlatformVersion>
+    <!-- BINDINGSGENERATOR : warning BG8604: Could not find top ancestor type 'LifecycleWatcher' for nested type 'Sentry.Android.LifecycleWatcher._1' -->
+    <!-- BG8604 Results in a missing API and for as long as we don't need that (and all Java/Android SDK is internal to this lib), it just doesn't matter: -->
+    <NoWarn>$(NoWarn);BG8604</NoWarn>
+    <SentryAndroidSdkVersion>5.3.0</SentryAndroidSdkVersion>
+    <AndroidLibsDirectory>$(MSBuildProjectDirectory)/../../libs/Android/</AndroidLibsDirectory>
+    <!-- Should be built in: https://docs.microsoft.com/en-us/dotnet/standard/frameworks#how-to-specify-a-target-framework but isn't available on .NET 6 RC2 -->
+    <DefineConstants>$(DefineConstants);NET6_0_ANDROID_OR_GREATER</DefineConstants>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Remove="Android/Transforms/*.xml" />
+    <TransformFile Include="Android/Transforms/*.xml" />
+    <!-- Gson will be dropped on sentry-android version 6.0.0 -->
+    <PackageReference Include="GoogleGson" Version="2.8.5" />
+    <!-- TODO: How to add JavaDocPaths for each package? -->
+    <AndroidLibrary Include="$(AndroidLibsDirectory)sentry-$(SentryAndroidSdkVersion).jar" />
+    <AndroidLibrary Include="$(AndroidLibsDirectory)/sentry-android-core-$(SentryAndroidSdkVersion).aar" />
+    <AndroidLibrary Include="$(AndroidLibsDirectory)/sentry-android-ndk-$(SentryAndroidSdkVersion).aar" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Android/*.cs" />
+  </ItemGroup>
+  <!-- Update Android SDK: Update SentryAndroidSdkVersion and run:
+    dotnet msbuild src/Sentry /t:DownloadSentryAndroidSdk /p:TargetFramework=net6.0-android -->
+  <Target Name="DownloadSentryAndroidSdk">
+    <RemoveDir Directories="$(AndroidLibsDirectory)" />
+    <DownloadFile SourceUrl="https://search.maven.org/remotecontent?filepath=io/sentry/sentry-android-core/$(SentryAndroidSdkVersion)/sentry-android-core-$(SentryAndroidSdkVersion).aar" 
```

Comment:
So we are going to retrieve the binary file from maven,  but we are bundling the binary here too

https://github.com/getsentry/sentry-dotnet/pull/1288/files#diff-c2f9235cb865a86e73577ce5850d351d005c29a2e589822fca2e14b4d8b56f41

Perhaps we should add those binaries to GitIgnore?

Reply:
It may be worth exploring the Xamarin.Build.Download package (version >= 0.11 necessary). It should have most of the logic already in a readily usable form.

Reply:
First take: https://github.com/filipnavara/sentry-dotnet/commit/1953501c744d333d197655d0978524c575c6cf47

Reply:
Nice! Thank you Filip